CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ if(CCACHE_FOUND AND ENABLE_CCACHE)
 	SET(CMAKE_CXX_FLAGS "${CCACHE_FLAGS} ${CMAKE_CXX_FLAGS}")
 endif()
 
+FIND_PACKAGE(Cereal REQUIRED)
```

Comment:
this should rather be like when looking for Eigen..... if it's available on the system use the system one, if not, then download it.